l10n_it_riba/tests/test_riba.py

```diff
@@ -239,6 +239,157 @@ def test_riba_flow(self):
         self.assertEqual(riba_list.state, "credited")
         self.assertEqual(riba_list.line_ids[0].state, "credited")
 
+    def test_riba_sbf_immediate_flow(self):
```

Comment:
Questo nuovo test contiene molto codice duplicato rispetto a quello ora presente in `test_riba_sbf_maturation_flow`. Non è possibile ottimizzarlo?